tests/test_flusso_detrito.py

```diff
@@ -0,0 +1,136 @@
+import pytest
+from qgis.core import QgsExpressionContextUtils, QgsMapLayer, QgsProject, QgsVectorLayer
+from qgis.testing import start_app
+from qgis.testing.mocked import get_iface
+
+from pzp.calculation import CalculationDialog
+from tests.utils import get_copy_path, get_data_path, get_expected_path
+
+start_app()
+import processing
+
+
+@pytest.fixture(scope="module", autouse=True)
+def initialize_processing():
+    print("\nINFO: Setting up processing")
+    from processing.core.Processing import Processing
+
+    Processing.initialize()
+
+
+@pytest.fixture(scope="module")
+def flusso_detrito_layer():
+    print("\nINFO: Get layer copy")
+    yield QgsVectorLayer(
```

Comment:
Why do you use yeld instead of return here?

Reply:
When using `pytest` `fixtures`, `yield` is the recommended way to go to have both `setUp` and `tearDown` in the same method (`tearDown` after the `yield` instruction).

However, in this particular case, when there are no subsequent lines, we should use `return`, I agree. It was probably a leftover. Thanks.

Reply:
Solved in https://github.com/opengisch/pzp/pull/20/commits/ffe3a7c5dfcc3a19c7d97a8a0d1a1bd001bbdf3b